templates/includes/login.html

```diff
@@ -1,4 +1,4 @@
-{% if user.is_authenticated() %}
+{% if (not hide_header_login) and user.is_authenticated() %}
```

Comment:
Let's turn this around and make it `{% if user.is_authenticated() and not hide_header_login %}`, much easier to read